lib/aggcat/base.rb

```diff
@@ -90,8 +99,9 @@ def iso8601(time)
 
     def parse_xml(data)
       return data if data.nil? || data.to_s.empty?
+      data.force_encoding('UTF-8')
       $stdout.puts(data) if @verbose
-      @parser ||= XmlHasher::Parser.new(snakecase: true, ignore_namespaces: true)
+      @parser ||= XmlHasher::Parser.new(snakecase: false, ignore_namespaces: true)
```

Comment:
This change is not backward compatible since might break code that relies on default non-snakecase format.    If you need non-snakecase format, then add a new option to switch between snakecase or non.